src/vs/workbench/contrib/terminal/test/common/environmentVariableService.test.ts

```diff
@@ -38,29 +42,30 @@ suite('EnvironmentVariable - EnvironmentVariableService', () => {
 			{ identifier: { value: 'ext2' } },
 			{ identifier: { value: 'ext3' } }
 		]);
+		instantiationService.stub(IHistoryService, historyService);
 
 		environmentVariableService = instantiationService.createInstance(TestEnvironmentVariableService);
 	});
 
 	test('should persist collections to the storage service and be able to restore from them', () => {
 		const collection = new Map<string, IEnvironmentVariableMutator>();
-		collection.set('A', { value: 'a', type: EnvironmentVariableMutatorType.Replace });
-		collection.set('B', { value: 'b', type: EnvironmentVariableMutatorType.Append });
-		collection.set('C', { value: 'c', type: EnvironmentVariableMutatorType.Prepend });
+		collection.set('A-key', { value: 'a', type: EnvironmentVariableMutatorType.Replace, scope: undefined, variable: 'A' });
```

Comment:
```suggestion
		collection.set('A-key', { value: 'a', type: EnvironmentVariableMutatorType.Replace,  variable: 'A' });
```

Reply:
if you change this to be `scope?: EnvironmentVariableScope`, you can delete all of these `scope: undefined` instances

https://github.com/microsoft/vscode/blob/0dc2f4867268ceea1c327215eb9c8f73d590cccf/src/vs/platform/terminal/common/environmentVariable.ts#L23

Reply:
For internal use making it optional will be cleaner